ci/get-engine.ps1

```diff
@@ -46,6 +50,10 @@ pushd "$($gdk_home)"
         }
     popd
 
+    ## Create an UnrealEngine symlink to the correct directory
+    Remove-Item "UnrealEngine" -ErrorAction ignore -Recurse -Force
```

Comment:
Two things - why is this necessary? Should this folder be fresh from the checkout?
And shouldn't we prepend $gdk_home to the path or use the $unreal_path below directly?

ie. cmd /c mklink /J $unreal_path "UnrealEngine-Cache\$($unreal_version)"

Reply:
In practice, yes it will be deleted - this is mainly so you can iterate on the script without having to delete it by hand in between each run.

IMO "UnrealEngine" is more clear than "$gdk_home", since it's clearly a directory in the cwd, whereas $gdk_home could be anywhere (even though its actually just the cwd).

Reply:
Thanks for clearing that up.

re: paths, I think because immediately after this we expect UnrealEngine to live in gdk_home, we should be making the symlink using it